core/local/local.go

```diff
@@ -67,13 +68,20 @@ func NewExecutionScheduler(runner lib.Runner, logger *logrus.Logger) (*Execution
 
 	executorConfigs := options.Execution.GetSortedConfigs()
 	executors := make([]lib.Executor, len(executorConfigs))
```

Comment:
hmm this is probably wrong now, after all, the `len` of the new slice could be shorter... we can still set `len(executorConfigs)` as the new slice `cap` though

Reply:
It's intended, and not wrong, the executors is filtered by the for loop below, without allocation if we use append.

Reply:
...
https://play.golang.org/p/Bnz-zVuSeA2

Reply:
@na-- You missed this line? https://github.com/loadimpact/k6/pull/1307/files#diff-b1b7b089b35559da00651021efdf45faR84

Reply:
But then let use slice cap ^^

Reply:
Ahh, you're right, sorry! :blush: Still, keeping track of the index and trimming the slice seems unnecessary, when you can just use `append()`...

Reply:
Fixed

Reply:
> Ahh, you're right, sorry! 😊 Still, keeping track of the index and trimming the slice seems unnecessary, when you can just use `append()`...

Yes, since when we only pre-allocate once, and the size of executors never greater than the size of executor configs.